ci-operator/jobs/quarkus-qe/quarkus-openshift-interop/quarkus-qe-quarkus-openshift-interop-main-periodics.yaml

```diff
@@ -19,7 +19,7 @@ periodics:
   name: periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.14-lp-interop-quarkus-interop-aws
   reporter_config:
     slack:
-      channel: '#quarkus-qe'
+      channel: '#team-quarkus-qe'
```

Comment:
@fedinskiy - Slack does not currently display channel '#team-quarkus-qe'. Does it need to be first created, is it a private channel, or is there another existing channel that can be used?

Reply:
Good catch @dfrazzette if the slack channel is not public then I don't believe the prow bot will be able to post a notification.

This is just public within the internal red hat slack workspace so its not public public.

Reply:
@dfrazzette this channel is existing, but private.

@ascerra the notifications worked just fine when this channel was private and named "quarkus-qe", so I expected they will work when this channel is private and named "team-quarkus-qe".